features.json

```diff
@@ -2,77 +2,102 @@
 	"$schema": "./features.schema.json",
 	"features": {
 		"bigInt": {
-			"description": "JS BigInt to Wasm i64 integration",
+			"description": "JS BigInt to Wasm i64 Integration",
 			"url": "https://github.com/WebAssembly/JS-BigInt-integration",
 			"phase": 5
 		},
+		"branchHinting": {
+			"description": "Branch Hinting",
+			"url": "https://github.com/WebAssembly/branch-hinting/blob/master/proposals/branch-hinting/Overview.md",
+			"phase": 5
+		},
 		"bulkMemory": {
-			"description": "Bulk memory operations",
+			"description": "Bulk memory Operations",
 			"url": "https://github.com/WebAssembly/bulk-memory-operations/blob/master/proposals/bulk-memory-operations/Overview.md",
 			"phase": 5
 		},
+		"customAnnotationSyntaxInTheTextFormat": {
+			"description": "Custom Text Format Annotations",
+			"url": "https://github.com/WebAssembly/annotations/blob/main/proposals/annotations/Overview.md",
+			"phase": 5
+		},
 		"customPageSizes": {
-			"description": "Custom page sizes",
+			"description": "Custom Page Sizes",
 			"url": "https://github.com/WebAssembly/custom-page-sizes/blob/main/proposals/custom-page-sizes/Overview.md",
-			"phase": 1
+			"phase": 2
+		},
+		"esmIntegration": {
+			"description": "ESM Integration",
+			"url": "https://github.com/WebAssembly/esm-integration",
+			"phase": 3
 		},
 		"exceptions": {
-			"description": "Exception handling",
+			"description": "Exception Handling",
```

Comment:
Ah, i just noticed that you describe that in the notes in this PR. My concern with this approach is that developers might think there is no supported EH option across the web, while in fact there is (i.e. until exnref is supported on all browsers, I expect that at least developers of C++ and any other toolchain that supports the existing EH will continue to use that).
Maybe we should figure out what flag options we want to use in emscripten and revisit this...
But anyway.... to avoid letting the perfect being the enemy of the good and stalling this too long, maybe let's just land this PR and then split it out in a separate PR. 
So maybe my question for you here is, how easy would it be with this code architecture to add another category to the table for deprecated features?

Reply:
I split it into a deprecated section now.